<launch>
  <!--node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" -->
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="360" />
	<param name="camera_info_url" value="package://usb_cam/camera_info/cam_data.yaml"/>
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <!--<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">-->
  <!--node name="image_view" pkg="image_view" type="image_view" respawn="false">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node-->
</launch>

